src/components/FormStart/index.js

```diff
@@ -124,7 +130,7 @@ const FormStart = ({form, hasActiveSubmission, onFormStart, onDestroySession}) =
 
 FormStart.propTypes = {
   form: Types.Form.isRequired,
-  hasActiveSubmission: PropTypes.bool.isRequired,
+  submission: Types.Submission,
```

Comment:
Why do we use both `Types` and `propTypes`?What is the difference?

Reply:
`Types` is defined by us to reuse types like submission/form: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/types/index.js